packages/components/src/playground/button.tsx

```diff
@@ -1,33 +1,20 @@
 import React from "react";
-import styled from "styled-components";
+import tw from "twin.macro";
 
 // Temporary props for button to test out css options for variants
 type ButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> & {
   variant?: "primary" | "secondary";
 };
 
-const ButtonComponent = styled.button<ButtonProps>`
-  border: none;
-  border-radius: 0.25rem;
-  color: white;
-  font-weight: bold;
-  padding: 0.5rem 1rem;
+const ButtonComponent = tw.button`
+  font-bold py-2 px-4 rounded text-white border-none
 `;
 
-const PrimaryButton = styled(ButtonComponent)`
-  background-color: #574eff;
-
-  &:hover {
-    background-color: #433bde;
-  }
+const PrimaryButton = tw(ButtonComponent)`
```

Comment:
Not necessary for this PR, but I actually liked the way you did variants for linaria with the one `tw.button` and the others were variant classes that were appended

Reply:
ooh ok, I did like that too. the `className` api might be a linaria-only thing but I can play a little

Reply:
updated, and I think it's now closer to what I was doing with linaria? lmk if I interpreted wrong